UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/UiChangeling.cs

```diff
@@ -0,0 +1,159 @@
+using Mirror;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using Systems.Antagonists;
+using TMPro;
+using UI.Core;
+using UI.Core.Action;
+using UnityEngine;
+
+namespace Changeling
+{
+	public class UiChangeling : MonoBehaviour
+	{
+		public ChangelingMain ChangelingMain;
+		public static UiChangeling instance;
```

Comment:
we use a singleton manager for this

Reply:
It's actually never used. Just forgot about remove that